AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md

```diff
@@ -0,0 +1,32 @@
+
+
+---
+AGP: N/A
+Title: Aragon Comprehensive Legal Review Program
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Aragon Comprehensive Legal Review Program
+
+## Description of desired Association policy change
+
+Fund 3-5 hours of legal council for every Aragon application shipped through the Nest or Flock program. This council would determine:
```

Comment:
Up to 3-5h

Reply:
Good call. Thanks

Reply:
New PR for this AGP can be found here:
- https://github.com/aragon/AGPs/pull/118